src/hooks/accounts/lib/getAccounts.tsx

```diff
@@ -0,0 +1,30 @@
+import { web3Accounts, web3Enable } from '@polkadot/extension-dapp';
+import { decodeAddress, encodeAddress } from '@polkadot/util-crypto';
+import { Account } from '../../../generated/graphql';
+import constants from '../../../constants';
+
+/**
+ * Used to fetch all accounts
+ */
+export const getAccounts = async (): Promise<Account[]> => {
+  // ensure we're connected to the polkadot.js extension
+  await web3Enable('basilisk-ui');
+
+  // get all the accounts from the polkadot.js extension
+  // return all retrieved accounts
+  const accounts = await web3Accounts();
+
+  // transform the returned accounts into the required entity format
+  return accounts.map((account) => {
+    const address = encodeAddress(
+      decodeAddress(account.address),
+      constants.basiliskAddressPrefix
```

Comment:
consider extracting this to single function toBasiliskFormattedAddress()

Reply:
also please move it to some shared utils file because other functions will need this as well